Tests/PackageLoading/ConventionTests.swift

```diff
@@ -96,13 +96,38 @@ class ConventionTests: XCTestCase {
         }
     }
 
+    func testCInTests() throws {
+        var fs = InMemoryFileSystem()
+        try fs.createEmptyFiles("/MyPackage/main.swift",
+                                "/MyPackage/Tests/MyPackage/abc.c")
+
+        PackageBuilderTester("/MyPackage", in: fs) { result in
+            result.checkModule("MyPackage") { moduleResult in
+                moduleResult.check(type: .executable, isTest: false)
+                moduleResult.checkSources("main.swift")
+                moduleResult.checkSources(root: "/MyPackage")
+            }
+
+            result.checkModule("MyPackageTestSuite") { moduleResult in
```

Comment:
Doesn't this test fail on Linux, for exactly the reasons this test is trying to test?